hummingbot/connector/exchange/paper_trade/__init__.py

```diff
@@ -1,6 +1,5 @@
 from typing import List, Callable
 from hummingbot.client.config.config_helpers import get_connector_class
-from hummingbot.connector.exchange.paper_trade.market_config import MarketConfig
 from hummingbot.connector.exchange.paper_trade.paper_trade_exchange import PaperTradeExchange
 from hummingbot.client.settings import AllConnectorSettings
```

Comment:
Since we are touching the import statements, let's take the opportunity to sort them correctly